spec/classes/init_spec.rb

```diff
@@ -608,6 +608,7 @@
     it { should contain_file('sshd_config').with_content(/^AcceptEnv L.*$/) }
     it { should_not contain_file('sshd_config').with_content(/^AuthorizedKeysFile/) }
     it { should_not contain_file('sshd_config').with_content(/^StrictModes/) }
+    it { should contain_file('sshd_config').with_content(/^MaxAuthTries 6/) }
```

Comment:
There are a handful of other spots that a line similar to this one should be in.